code/API_definitions/qos-profiles.yaml

```diff
@@ -124,20 +128,20 @@ paths:
         "503":
           $ref: "#/components/responses/Generic503"
 
-  /qos-profiles/{name}:
+  /retrieve-qos-profile/{name}:
```

Comment:
I thought we were going to keep `GET /qos-profiles/{name}`. That would be the expected name for such an endpoint.